lib/util/check-prefer-global.js

```diff
@@ -5,16 +5,21 @@
 "use strict"
 
 const { ReferenceTracker } = require("@eslint-community/eslint-utils")
-const extendTrackmapWithNodePrefix = require("./extend-trackmap-with-node-prefix")
+
+/**
+ * @typedef {Object} TraceMap
```

Comment:
```suggestion
 * @typedef TraceMap
```

When using `@property` no type is needed in `@typedef`, its implied that its an object